client/stylesheets/_toCat.scss

```diff
@@ -1130,15 +1130,20 @@ HELP | ABOUT | FAQ | SAFETY | PRIVACY | CONTACT **/
 
 /****** CONTACT ******/
 
+.contactform {
+    max-width: 400px !important;
+    margin: 0 auto;
+}
+
 .contactname input, .contactsubject input, .contactemail input { 
     padding: 10px 12px !important;
-    width: 400px !important;
+    width: 100% !important;
     border-radius: 4px !important;
     border: 1px solid #d3d3d3 !important;
 }
 
 .contacttext textarea {
-    width: 400px !important;
+    width: 100% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border, border-radius, height, padding, width